src/command_modules/azure-cli-batch/azure/cli/command_modules/batch/_params.py

```diff
@@ -7,7 +7,7 @@
 
 from knack.arguments import CLIArgumentType
 
-from azure.mgmt.batch.models import AccountKeyType
+from azure.mgmt.batch.models.batch_management_client_enums import AccountKeyType
```

Comment:
@lmazuel 
Models should be referenced at the `azure.mgmt.batch.models` only. Lower level namespace should not be referenced.